app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics } = require("./controllers/controllers");
+
+const app = express();
+app.use(express.json());
```

Comment:
Line 5 isn't currently being utilised by any of your endpoints so at this point in development it's unnecessary code in your `app.js` file